proarc-webapp/src/main/java/cz/cas/lib/proarc/webapp/client/widget/DigitalObjectChildrenEditor.java

```diff
@@ -669,9 +736,24 @@ public void execute() {
                             childrenListGrid.scrollToRow(recordIndex);
                         }
                     });
+
+                    if (dialog != null) {
+                        dialog.destroy();
+                    }
+                }
+
+                if (response.getStatus() == RPCResponse.STATUS_VALIDATION_ERROR) {
+                    if (paramsForm != null) {
+                        Map errors = response.getErrors();
+
+                        paramsForm.setErrors(errors, true);
+                        //request.setWillHandleError(true);
```

Comment:
Co to dělá a proč je to zakomentované?

Reply:
Šlo o artefakt předchozího způsobu implementace, handle errors se nyní nastavuje pouze pokud jde o požadavek s nastavením UUID, viz dsRequest.setWillHandleError(true); o pár řádků výš.

Jde o to, že pokud selže požadavek na backend na vytvoření objektu s UUID (např objekt s daným UUID už existuje), tak se chyby "chopí" frontend tím, že chybu promítne do dialogového okna.

Reply:
Odebráno. Viz f76b506296280f2bb1f9d43a66618184d54016ff